github/messages_test.go

```diff
@@ -220,3 +220,17 @@ func TestParseWebHook(t *testing.T) {
 		}
 	}
 }
+
+func TestDeliveryID(t *testing.T) {
+	id := "8970a780-244e-11e7-91ca-da3aabcb9793"
+	req, err := http.NewRequest("POST", "http://localhost", nil)
+	if err != nil {
+		t.Fatalf("DeliveryID: %v", err)
+	}
+	req.Header.Set("X-Github-Delivery", id)
+
+	got := DeliveryID(req)
+	if got != id {
+		t.Errorf("DeliveryID(%#v) = %#v, want %#v", req, got, id)
```

Comment:
nit: since `got` and `id` are both strings, you may want to use `%q` for these values instead of `%#v`.

`t.Errorf("DeliveryID(%#v) = %q, want %q", req, got, id)`